SU-SWS/saa_victoria_subtheme
# Adapt 552  mobile spacing v3

# READY FOR REVIEW

# Summary
- Tweaking mobile spacing numbers and double-checking media + text area faux cards

# Review By (Date)
- august 31

# Urgency
- medium, to wrap up the project

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Test spacing between cards from paragraph's bottom margin:  https://saauserguide.sites.stanford.edu/site-building-guide/saa-components-stanford-sites
4. Check and make sure there is no overlap or bad spacing at mobile /tablet / or desktop sizes with media + text area faux paragraphs:
https://saauserguide.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/cards/large-horizontal-21-card

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-552
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
